components/supervisor/pkg/supervisor/supervisor.go

```diff
@@ -778,7 +784,7 @@ func runIDEReadinessProbe(cfg *Config, ideConfig *IDEConfig, ide IDEKind) (deskt
 		var (
 			schema = defaultIfEmpty(ideConfig.ReadinessProbe.HTTPProbe.Schema, "http")
 			host   = defaultIfEmpty(ideConfig.ReadinessProbe.HTTPProbe.Host, "localhost")
-			port   = defaultIfZero(ideConfig.ReadinessProbe.HTTPProbe.Port, cfg.IDEPort)
+			port   = defaultIfZero(ideConfig.ReadinessProbe.HTTPProbe.Port, desktopIDEPort)
```

Comment:
here it should be `cfg.IDEPort` for web, but `desktopIDEPort` for desktop?

Reply:
maybe let's rename method, it was confusing for me from GitHub, i.e. `runDesktopIDEReadinessProbe` or pass it as an argument?